.travis.yml

```diff
@@ -129,8 +129,16 @@ script:
           mkdir for_testing
           cd for_testing
           cp ../.coveragerc .
+          PASS=true
           nosetests --with-doctest --with-coverage --cover-package nibabel nibabel
+          if [ "$?" != "0" ]; then
+              PASS=false;
+          fi
           pytest -v ../nibabel/tests/test_affines.py ../nibabel/tests/test_volumeutils.py
+          if [ "$?" != "0" ]; then
+              PASS=false;
+          fi
+          $PASS
```

Comment:
@djarecka Just a note that your pytest tests are masking nosetest failures. Can you please cherry-pick 2716f8d600cb511eddd7d314854387f008e6b17a or rebase onto this branch?

Reply:
sorry for this, maybe you can simply remove pytest from travis for now, and just wait for #840

Reply:
Sure.